### PR TITLE
Add DLS B24 packages

### PR DESCRIPTION
**Pull request by thomasmfish**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by thomasmfish** [comment]
Failing on Windows build. Not quite sure why yet - will look into it tomorrow.

**Comment by thomasmfish** [comment]
Does the Azure CI have a user desktop that can be written to? If not, maybe that's where it is freezing. Let's find out!

**Comment by thomasmfish** [comment]
Well, that seemed to do it!

**Comment by thomasmfish** [comment]
@conda-forge/staged-recipes

File: recipes/StitchM/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - pip
+    - numpy >=1.17.4
```
Comment (scopatz) [inline recipes/StitchM/meta.yaml:26]: Are you linking against NumPy? If so, please add a C compiler and follow the directions https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

File: recipes/txrm2tiff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - {{ name }} = {{ name }}.scripts.commandline:main
+  skip: True  # [py<35]
+
+requirements:
+  
+  build:
+    - python
+    - setuptools
```
Comment (scopatz) [inline recipes/txrm2tiff/meta.yaml:23]: These should not be in build.

File: recipes/StitchM/meta.yaml (hunk 1 above)
Comment (thomasmfish) [reply recipes/StitchM/meta.yaml]: I had an import that was breaking a script being created without numpy in host. I've fixed it now and moved numpy to run requirements.

File: recipes/txrm2tiff/meta.yaml (hunk 2 above)
Comment (thomasmfish) [reply recipes/txrm2tiff/meta.yaml]: Okay, I've removed those

**Pull request merged**